app/views/layouts/application.html.erb

```diff
@@ -47,7 +47,8 @@
         <div class="main-menu">
           <ul id="side-main-menu" class="side-menu list-unstyled">
             <li class="<%= @all_leads_active %>"><a href="/"> <i class="icon-home"></i><span>All Leads</span></a></li>
-            <li class="<%= @outbound_mode_active %>"> <a href="/next" data-turbolinks="false"><i class="icon-form"></i><span>Outbound Mode</span></a></li>
+            <li class="<%= @lead %>"><a href="/leads/new"><i class="icon-flask"></i><span>New Leads</span></a></li>
```

Comment:
@ProfBologna, @xthegoodguy I think this is still in progress, right? There's another version that has the variable that activates the `active` class? Should I approve this or wait for the next PR?